spec/api/datastore/v2/reviewing/complete_application_spec.rb

```diff
@@ -31,21 +31,15 @@
         application.update!(review_status: :assessment_completed, reviewed_at: 1.week.ago)
       end
 
-      it 'raises a 409 error' do
-        api_request
-        expect(response).to have_http_status :conflict
-      end
+      it_behaves_like 'raises a 409 error'
```

Comment:
Nice

Reply:
Wonder if we can reword this to a sentence` it_behaves_like 'raises a 409 error'` e.g. ` it_behaves_like 'an error that raises a 409 status code'`